api/src/components/additionalInformation/__tests__/createAdditionalInformation.test.ts

```diff
@@ -0,0 +1,114 @@
+import * as testUtils from 'lib/testUtils';
+
+describe('Create a piece of additional information', () => {
+    beforeAll(async () => {
+        await testUtils.clearDB();
+        await testUtils.testSeed();
+    });
+
+    test('User can add additional information to their DRAFT publication version', async () => {
+        const createPayload = {
+            title: 'Example title',
+            url: 'https://jisc.ac.uk'
+        };
+        const createAdditionalInformation = await testUtils.agent
+            .post('/publication-versions/publication-problem-draft-v1/additional-information')
+            .query({ apiKey: '000000005' })
+            .send(createPayload);
+
+        expect(createAdditionalInformation.status).toEqual(200);
+        expect(createAdditionalInformation.body).toMatchObject(createPayload);
+    });
+
+    test('User cannot add additional information to a non-existent publication version', async () => {
+        const createAdditionalInformation = await testUtils.agent
+            .post('/publication-versions/made-up-publication-version-id/additional-information')
+            .query({ apiKey: '000000005' })
+            .send({
+                title: 'Example title',
+                url: 'https://jisc.ac.uk'
+            });
+
+        expect(createAdditionalInformation.status).toEqual(404);
+        expect(createAdditionalInformation.body.message).toEqual('This publication version does not exist');
+    });
+
+    test('User cannot add additional information to their LIVE publication version', async () => {
+        const createAdditionalInformation = await testUtils.agent
+            .post('/publication-versions/publication-problem-live-v1/additional-information')
+            .query({ apiKey: '123456789' })
+            .send({
+                title: 'Example title',
+                url: 'https://jisc.ac.uk'
+            });
+
+        expect(createAdditionalInformation.status).toEqual(403);
```

Comment:
Not 100% sold on which status this should be 🤔 

Leaning towards 400 as with the others, the request is invalid, attempting to create additional information against a publication in an invalid state.

Reply:
Have changed this and some other responses across the application that were using 403 in this type of situation.